vividus-extension-selenium/src/main/java/org/vividus/steps/ui/GenericWaitSteps.java

```diff
@@ -132,6 +140,60 @@ public String toString()
         });
     }
 
+    /**
+     * Waits for an <b><i>element</i></b> with the specified <b>locator</b> to stop moving
+     * @param locator locator to locate element
+     */
+    @When("I wait until element located by `$locator` stops moving")
+    public void waitForElementStopsMoving(Locator locator)
+    {
+        uiContext.getOptionalSearchContext().ifPresent(context -> {
+            baseValidations.assertElementExists("The element to validate stop of its movement",
+                    context, locator).ifPresent(webElement -> {
+                        locator.getSearchParameters().setWaitForElement(false);
+                        IExpectedSearchContextCondition<Boolean> condition = new IExpectedSearchContextCondition<>()
+                        {
+                            private Point lastPosition;
+                            private WebElement movingWebElement = webElement;
+
+                            @Override
+                            public Boolean apply(SearchContext context)
+                            {
+                                try
+                                {
+                                    Point currentPosition = movingWebElement.getLocation();
+                                    boolean samePosition = currentPosition.equals(lastPosition);
+                                    if (!samePosition)
+                                    {
+                                        LOGGER.info("The element is moved at '{}'", currentPosition);
+                                    }
+                                    lastPosition = currentPosition;
+                                    return samePosition;
+                                }
+                                catch (StaleElementReferenceException e)
+                                {
+                                    List<WebElement> movingWebElements = searchActions.findElements(context, locator);
```

Comment:
shouldn't we stop waiting after the 2 StaleElementReferenceException? otherwise there could be the chance we'll never catch element movement, but we'll just wait and check some flashing element

Reply:
added `boolean elementPresent` to validate it